misc/config_tools/schema/types.xsd

```diff
@@ -50,6 +50,16 @@
   </xs:restriction>
 </xs:simpleType>
 
+<xs:simpleType name="MaxVMNumType">
+  <xs:annotation>
+    <xs:documentation>Integer from 1 to 8.</xs:documentation>
+  </xs:annotation>
+  <xs:restriction base="xs:integer">
+    <xs:minInclusive value="1" />
+    <xs:maxInclusive value="8" />
+  </xs:restriction>
+</xs:simpleType>
```

Comment:
Creating a separate integer type solely due to a different range seems an overkill to me. You can embed the type in where it is used directly. See https://www.w3schools.com/xml/el_restriction.asp for examples.

We may need to clean up the types here, but in a future PR probably.